server/src/dismissable-flags/dismissable-flags.service.ts

```diff
@@ -0,0 +1,56 @@
+import { Injectable } from '@nestjs/common';
+import { ApiTags } from '@nestjs/swagger';
+import { PrismaService } from '../prisma.service';
+
+export enum DismissableFlags {
```

Comment:
@gregogun I think it would be better to use a string enum there, would make the code simpler

Reply:
ok will update it 👍 would we need to update the enum on the client too?